mutable_tree.go

```diff
@@ -437,6 +557,50 @@ func (tree *MutableTree) LoadVersionForOverwriting(targetVersion int64) (int64,
 	return latestVersion, nil
 }
 
+func (tree *MutableTree) enableFastStorageAndCommitIfNotEnabled() (bool, error) {
+	if tree.ndb.isFastStorageEnabled() {
+		return false, nil
+	}
+
+	if err := tree.enableFastStorageAndCommit(); err != nil {
+		tree.ndb.storageVersion = defaultStorageVersionValue
+		return false, err
+	}
+	return true, nil
+}
+
+func (tree *MutableTree) enableFastStorageAndCommitLocked() error {
+	tree.mtx.Lock()
+	defer tree.mtx.Unlock()
+	return tree.enableFastStorageAndCommit()
+}
+
+func (tree *MutableTree) enableFastStorageAndCommit() error {
```

Comment:
Can we add some fmt.Printlns here, so people know why things are taking awhile?

Reply:
Will do. That method is also called in `LoadVersionForOverwriting`. Is that acceptable to print in that case as well?

Reply:
Enabled prints in debug mode for now. I'm a bit unsure about regular `Println` in production. Please let me know if that's what you meant